src/System.Private.DisabledReflection/src/System.Private.DisabledReflection.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition="'$(AssemblyName)' == ''">
+    <AssemblyName>System.Private.DisabledReflection</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <IsDotNetFrameworkProductAssembly>true</IsDotNetFrameworkProductAssembly>
+    <ProjectGuid>{4BD898B4-D652-4F61-B3ED-F3C847D77E25}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(IsProjectNLibrary)' != 'true'">
+    <ProjectReference Include="..\..\System.Private.CoreLib\src\System.Private.CoreLib.csproj">
+      <Aliases>global,System_Private_CoreLib</Aliases>
```

Comment:
I am deleting these aliases in https://github.com/dotnet/corert/pull/7216 . They are no longer needed.